src/patternfly/layouts/Gallery/styles.scss

```diff
@@ -1,7 +1,24 @@
+@import "../../patternfly-utilities";
+
 .pf-l-gallery {
-  display: grid;
-  grid-gap: 1rem; //revisit this value
-  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
-  padding: 0;
-  margin: 0;
+  $pf-l-gallery--GridGap: 1rem !default;
+  $pf-l-gallery--GridTemplateColumns: repeat(auto-fill, minmax(250px, 1fr)) !default;
```

Comment:
Maybe this is stupid, but what if we only make the `250px` part a variable?
also, we should check with @kybaker to know what that number should be and maybe use it in rems?

Reply:
👍 agreed